tests/src/whisk/core/cli/test/WskCoreBasicTests.scala

```diff
@@ -0,0 +1,210 @@
+
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.BeforeAndAfterAll
+
+import common.RunWskAdminCmd
+import common.TestHelpers
+import common.TestUtils
+import common.TestUtils.ANY_ERROR_EXIT
+import common.TestUtils.FORBIDDEN
+import common.TestUtils.NOT_FOUND
+import common.TestUtils.TIMEOUT
+import common.Wsk
+import common.WskProps
+import common.WskTestHelpers
+import spray.json.DefaultJsonProtocol._
+import spray.json.pimpAny
+import whisk.core.entity.Subject
+import whisk.core.entity.WhiskPackage
+
+@RunWith(classOf[JUnitRunner])
+class WskCoreBasicTests
+    extends TestHelpers
+    with WskTestHelpers
+    with BeforeAndAfterAll {
+
+    val originWskProps = WskProps()
+    val wsk = new Wsk(usePythonCLI = false)
+    val samplePackage = "samplePackage"
+    val sampleAction = s"$samplePackage/sampleAction"
+    val sampleFeed = s"$samplePackage/sampleFeed"
+    val wskadmin = new RunWskAdminCmd {}
+
+    val otherNamespace = Subject().toString()
+    val create = wskadmin.cli(Seq("user", "create", otherNamespace))
+    val otherAuthkey = create.stdout.trim
+    implicit val otherWskProps = WskProps(namespace = otherNamespace, authKey = otherAuthkey)
+
+    override def afterAll() = {
+        withClue(s"failed to delete temporary namespace $otherNamespace") {
+            wskadmin.cli(Seq("user", "delete", otherNamespace)).stdout should include("Subject deleted")
+        }
+    }
+
+    behavior of "Wsk CLI"
+
+    it should "reject deleting action in shared package not owned by authkey" in withAssetCleaner(otherWskProps) {
+        (wp, assetHelper) =>
+            assetHelper.withCleaner(wsk.pkg, samplePackage) {
+                (pkg, _) =>
+                    pkg.create(samplePackage, parameters = Map("a" -> "A".toJson), shared = Some(true),
+                               update = true)
+            }
+            assetHelper.withCleaner(wsk.action, sampleAction) {
+                val file = Some(TestUtils.getTestActionFilename("empty.js"))
+                (action, _) => action.create(sampleAction, file, kind = Some("nodejs"), shared = Some(true))
+            }
+            val fullyQualifiedActionName = "/" + otherNamespace + "/" + sampleAction
+            wsk.action.get(fullyQualifiedActionName)(originWskProps)
+            wsk.action.delete(fullyQualifiedActionName, expectedExitCode = FORBIDDEN)(originWskProps)
+    }
+
+    it should "reject create action in shared package not owned by authkey" in withAssetCleaner(otherWskProps) {
+        (wp, assetHelper) =>
+            assetHelper.withCleaner(wsk.pkg, samplePackage) {
+                (pkg, name) => pkg.create(name, shared = Some(true))
+            }
+            val fullyQualifiedActionName = s"/$otherNamespace/notallowed"
+            val file = Some(TestUtils.getTestActionFilename("empty.js"))
+            assetHelper.withCleaner(wsk.action, fullyQualifiedActionName, confirmDelete = false) {
```

Comment:
move this up to the previous test? (this would save having to create a new package).
if you do, rename that test appropriately.